scalanativelib/src/mill/scalanativelib/ScalaNativeModule.scala

```diff
@@ -114,7 +114,7 @@ trait ScalaNativeModule extends ScalaModule { outer =>
 
   override def scalacPluginIvyDeps: T[Agg[Dep]] = T {
     super.scalacPluginIvyDeps() ++ Agg(
-      ivy"org.scala-native:::nscplugin:${scalaNativeVersion()}"
+      ivy"org.scala-native:::nscplugin:${scalaNativeVersion()}".withPlatformed(false)
```

Comment:
Compiler plugins usually are JVM dependencies. Are there use cases for platformed dependencies with `:::`?
Are there Scala.js or Scala Native libraries that are published with the full Scala version?

Reply:
I think this is rendered moot by moving the `withPlatformed` call into `bindDependencies`, so a user no longer needs to care about this at all when specifying their `ivy"..."` strings